code/modules/multiz/movement.dm

```diff
@@ -355,6 +355,17 @@
 /obj/structure/catwalk/CheckFall(var/atom/movable/falling_atom)
 	return falling_atom.fall_impact(src)
 
+/atom/movable/proc/can_fall_to(turf/landing)
```

Comment:
We already have this. https://github.com/VOREStation/VOREStation/blob/1331236464c830b93f09b3659c40d171d712731b/code/modules/multiz/movement.dm#L326-L337

Reply:
Removed.